lib/avromatic/schema_registry_patch.rb

```diff
@@ -0,0 +1,23 @@
+require 'avro_turf'
+require 'avro_turf/schema_registry'
+
+AvroTurf::SchemaRegistry.class_eval do
+  # Override register to first check if a schema is registered by fingerprint
+  def register(subject, schema)
+    raise 'schema must be an Avro::Schema' unless schema.is_a?(Avro::Schema)
```

Comment:
Comparing to this implementation, https://github.com/salsify/avro_turf/blob/salsify-master/lib/avro_turf/schema_registry.rb#L19. Why do we decide to raise an error here?